frontend/src/views/activity/Activity.vue

```diff
@@ -40,16 +67,24 @@ Displays a single activity
             {{ activity.title }}
           </a>
         </v-toolbar-title>
-        <div v-if="editActivityTitle" class="mx-2" style="flex-grow: 1">
+        <div
+          v-if="editActivityTitle"
+          class="mx-2 d-flex align-center"
+          style="flex-grow: 1"
+        >
           <api-text-field
             :uri="activity._meta.self"
             fieldname="title"
             :disabled="layoutMode"
             dense
             autofocus
             :auto-save="false"
+            class="flex-grow-1"
             @finished="editActivityTitle = false"
           />
+          <v-btn text v-on="on" @click="editActivityTitle = false"
```

Comment:
Now we have two Cancel-Buttons... 
![image](https://user-images.githubusercontent.com/449555/200133781-e5864a7a-50d4-4795-8a51-a32db00f9629.png)

Could we avoid the extra button but configure it in a way the cancel-icon already appears at the beginning (not only when dirty)? Also the extra cancel-button is very easy to miss visually, so I'd rather have it inside the textfield to connect it logically directly to the textfield itself.